knowledge-base/architectural-decisions-records/v4-api.md

```diff
@@ -251,7 +251,7 @@ because it replaces the existing data with the payload.
   - delete entities (`@removed`)
   - delete links (`reason = deleted`)
   - add links (with id/key)
-  - update entities (with id/key + Partial<Entity>)
+  - update entities (with id/key + Partial<EntityV2>)
```

Comment:
I would revert all changes, because for this MD files I think it is not relevant if in some code snipped the EntityV4 or Entity is given. I think we understand what is meant.